tests/OmniSharp.DotNetTest.Tests/GetTestStartInfoFacts.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Threading.Tasks;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace OmniSharp.DotNetTest.Tests
+{
+    public class GetTestStartInfoFacts : AbstractGetTestStartInfoFacts
+    {
+        public GetTestStartInfoFacts(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        public override bool UseLegacyDotNetCli { get; } = false;
+
+        [Fact]
+        public async Task RunXunitTest()
+        {
+            await GetDotNetTestStartInfoAsync(
+                XunitTestProject,
+                methodName: "Main.Test.MainTest.Test",
+                testFramework: "xunit");
+        }
+
+        // NUnit does not work with .NET CLI RTM yet. https://github.com/nunit/dotnet-test-nunit/issues/108
+        // When it does, the NUnitTestProject should be updated and the tests below re-enabled.
+
+        //[Fact]
```

Comment:
a minor nit: you could say `[Fact(Skip = "{reason}"]`. This way the test will not be forgotten - it will always be reported as being there, but explicitly skipped.